activity_browser/layouts/tabs/activity.py

```diff
@@ -171,6 +185,10 @@ def connect_signals(self):
         signals.database_changed.connect(self.populate)
         signals.parameters_changed.connect(self.populate)
         # signals.activity_modified.connect(self.update_activity_values)
+        signals.activity_table_toggled.connect(self.activity_tables)
+
+    def activity_tables(self) -> None:
```

Comment:
should we maybe have either a description or a slightly better function name here? I'm not sure what this does.

Reply:
will do, sorry that was really bad practice

Reply:
So all of your comments were on part of the code that was dealing with issue #270 and was decided against implementing, hence the marking of these as being resolved!